pandas/core/frame.py

```diff
@@ -550,15 +550,33 @@ def _get_axes(N, K, index=index, columns=columns):
     @property
     def axes(self):
         """
-        Return a list with the row axis labels and column axis labels as the
-        only members. They are returned in that order.
+        Return a list representing the row axis labels and column axis labels
+        as the only members. They are returned in that order.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.axes
+        [RangeIndex(start=0, stop=2, step=1), Index(['coll', 'col2'],
+        dtype='object')]
         """
         return [self.index, self.columns]
 
     @property
     def shape(self):
         """
         Return a tuple representing the dimensionality of the DataFrame.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.shape
+        (2, 2)
+
+        >>> df = pd.DataFrame({'col0': [1, 2, 3], 'col2': [4, 5, 6],
+        ...                    'col3': [7, 8, 9]})
+        >>> df.shape
+        (3, 3)
```

Comment:
Can you make an example where the number of columns and number of rows is not identical?

Reply:
Okay, I will make it.